contracts/interfaces/ICube3Integration.sol

```diff
@@ -26,7 +26,7 @@ interface ICube3Integration {
     ///      has been deployed.
     /// @param self The contract address (this) of the new integration.
     /// @param admin The default securityAdmin account at the time of deployment, ie the deployer.
```

Comment:
should remove "ie the deployer" as that's not true with the upgradeable contract, or clarify in the comment.